codespell_lib/data/dictionary.txt

```diff
@@ -5241,6 +5242,7 @@ certifiates->certificates
 certifiating->certificating
 certifiation->certification
 certifiations->certifications
+certific->certificate
```

Comment:
Again could be certification or certify (possibly) etc.

Reply:
Done, see:
https://github.com/codespell-project/codespell/blob/f78823b28c2e8f7b7b3b622402954a784aa98548/codespell_lib/data/dictionary.txt#L5245